OrbitGl/TimeGraph.cpp

```diff
@@ -546,6 +546,25 @@ void TimeGraph::UpdatePrimitives(bool a_Picking) {
           Color grey(g, g, g, 255);
           static Color selectionColor(0, 128, 255, 255);
           Color col = GetThreadColor(timer.m_TID);
+
+          // We disambiguate the different types of GPU activity based on the
+          // string that is displayed on their timeslice.
+          if (timer.m_Type == Timer::GPU_ACTIVITY) {
+            float coeff = 1.0f;
+            std::string gpu_stage = string_manager_->Get(timer.m_UserData[0]).value_or("");
+            if (gpu_stage == "sw queue") {
```

Comment:
maybe extract those and make them constants?

Reply:
Done.